components/automate-ui/src/app/modules/desktop/desktop-detail/desktop-detail.component.html

```diff
@@ -22,10 +22,16 @@ <h3>Check-in History</h3>
           <span>Last {{ (checkinNumDays === 15 ? 2 : 4) }} Weeks</span>
           <chef-icon>expand_more</chef-icon>
         </chef-button>
-        <chef-button secondary>
+        <chef-button secondary (click)="toggleDownloadDropdown()" class='download-dropdown-toggle'>
           <span>Download</span>
           <chef-icon>expand_more</chef-icon>
         </chef-button>
+          <chef-dropdown class='download' [visible]="downloadDropdownVisible">
+            <chef-click-outside omit="download-dropdown-toggle" (clickOutside)="closeDownloadDropdown()">
+              <div (click)="onDownloadCheckInHistory('json')">JSON</div>
+              <div (click)="onDownloadCheckInHistory('csv')">CSV</div>
```

Comment:
I feel like these should probably be `li`'s in an unordered list.  Even though there are only two options, they are still technically a list.

Also, They should likely be `button` or `a` elements.  They are clickable and perform a function.  OR they should have appropriate aria attributes.  In their current state, they are not a11y and `button`/`a` can accomplish that natively for us.